src/software/ai/hl/stp/tactic/test_tactics/move_test_tactic.cpp

```diff
@@ -31,14 +31,11 @@ double MoveTestTactic::calculateRobotCost(const Robot &robot, const World &world
     return std::clamp<double>(cost, 0, 1);
 }
 
-void MoveTestTactic::calculateNextIntent(IntentCoroutine::push_type &yield)
+void MoveTestTactic::calculateNextAction(ActionCoroutine::push_type &yield)
 {
     do
     {
-        yield(std::make_unique<MoveIntent>(this->robot->id(), this->destination,
-                                           Angle::zero(), 0.0, 0, DribblerEnable::OFF,
-                                           MoveType::NORMAL, AutokickType::NONE,
-                                           BallCollisionType::AVOID));
+        yield(std::make_shared<MoveAction>());
```

Comment:
This should still return a proper MoveIntent?

Reply:
- this is a tactic, so it yields an action now?
- what do you mean by "proper"? This is intended to be a test tactic, so it's implementation is informed by the test requirements, which just require this. I think adding anything extra without have specific reasons for doing some from the test side is more confusing if anything.

Reply:
Oh sorry I totally misread this, thought it was returning an empty / null shared ptr. No issues here.